cv_next/app/login/page.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable @next/next/no-async-client-component */
+"use client"
+import { createClientComponentClient } from '@supabase/auth-helpers-nextjs'
+import { Auth } from '@supabase/auth-ui-react'
+import { ThemeSupa } from '@supabase/auth-ui-shared'
+
+export default async function Page() {
+    //const supabase = SupabaseHelper.getSupabaseInstance()
```

Comment:
Just adding a comment here so that you won't forget to delete dangling comments

Reply:
this is a test to see functionality

Reply:
another test